operations/network/management_network/fresh_install.md

```diff
@@ -2,11 +2,12 @@
 
 This is either a first time install or the previous CSM was wiped and requires a new install.
 
-:exclamation: All of these steps should be done using an out of band connection. This process is disruptive and will require downtime :exclamation: 
-
+:exclamation: All of these steps should be done using an out of band connection. This process is disruptive and will require downtime :exclamation:
 
 1. [Update management network firmware](firmware/update_management_network_firmware.md)
     - Upgrade switch firmware to specified firmware version.
+1. [Backup custom config](backup_custom_config.md)
+    - If the system had a previous version of CSM on it, you need to backup all custom configuration and credential configuration.
```

Comment:
Reword to remove use of "you":

If the system had a previous version of CSM on it, back up all of the custom configuration and credential configuration.